testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -461,6 +466,14 @@
   end
 end
 
+Then(/^the table row for "([^"]*)" should( not)? contain "([^"]*)" icon$/) do |row, should_not, icon_class|
+  xpath_query = "//div[@class=\"table-responsive\"]/table/tbody/tr[.//*[contains(.,'#{row}')]]"
+  within(:xpath, xpath_query) do
+    func = should_not ? :has_no_css? : :has_css?
+    raise "xpath: #{xpath_query} has no content #{icon_class}" unless send(func, "i[class*='#{icon_class}']", wait: DEFAULT_TIMEOUT)
```

Comment:
I think you can pacify Rubocop replacing `send` with `Capybara.public_send`.
I understand you want to avoid code duplication; still the raised error will be
wrong 50% of the times.

In this case, I would write just the 2-branch form:
```
if should_not
  raise "xpath: #{xpath_query} has content #{icon_class}" if has_css?("i[class*='#{icon_class}']", wait: DEFAULT_TIMEOUT)
else
  raise "xpath: #{xpath_query} has no content #{icon_class}" unless has_css?("i[class*='#{icon_class}']", wait: DEFAULT_TIMEOUT)
end
```

Reply:
Yeah, I was wondering, why was that (I don't know Ruby much). Ok, I'll re-write it accordingly :+1: